src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
Well, it is easy to test. If you don't bump the version, and you can still build nqp, then it's fine.